modules/ai/assets/js/editor/ai-behavior.js

```diff
@@ -99,5 +122,41 @@ export default class AiBehavior extends Marionette.Behavior {
 		$wrap.after(
 			$button,
 		);
+
+		const controlType = this.view.model.get( 'type' );
+		const promotionTexts = this.getPromotionTexts( controlType );
+		if ( ! promotionTexts ) {
+			return;
+		}
+
+		const editorSessionValue = sessionStorage.getItem( 'ai_promotion_introduction_editor_session_key' );
+		if ( ! editorSessionValue || editorSessionValue !== EDITOR_SESSION_ID ) {
+			sessionStorage.setItem( 'ai_promotion_introduction_editor_session_key', EDITOR_SESSION_ID );
+		} else {
+			return;
+		}
+		setTimeout( () => {
+			const rootBox = $button[ 0 ].getBoundingClientRect();
+			if ( ! rootBox || 0 === rootBox.width || 0 === rootBox.height ) {
+				return;
+			}
+
+			const rootElement = document.createElement( 'div' );
+			document.body.append( rootElement );
+
+			const { unmount } = ReactUtils.render( (
+				<AiPromotionInfotipWrapper
```

Comment:
Seems that there is a certain offset with the info-tip pointer, have you seen it as well?